ansible/ansible-ai-connect-service
# add darwin x86_64 support to requirements.in

This PR does not need a corresponding Jira item.

## Description
This PR allows a clean `pip install -r requirements.in` on intel mac.

## Testing

### Steps to test
1. Pull down the PR on an intel mac.
2. pip install -r requirements.in
3. Confirm torch and torchvision successfully install

### Scenarios tested
Steps above

## Production deployment
